src/equations/shallow_water_1d.jl

```diff
@@ -547,6 +582,30 @@ end
     return waterheight(u, equations) * pressure(u, equations)
 end
 
+"""
+    calc_wavespeed_roe(u_ll, u_rr, direction::Integer,
+                       equations::ShallowWaterEquations1D)
+
+Calculate Roe-averaged velocity `v_roe` and wavespeed `c_roe = sqrt{g h_roe}`
```

Comment:
```suggestion
Calculate Roe-averaged velocity `v_roe` and wavespeed `c_roe = sqrt(g * h_roe)`.
```
Please use code in code font (as suggested above) or mark it as math and use LaTeX.